### PR TITLE
Add Expression parser.

### LINKED ISSUE #40: Support "Expressions"
https://webassembly.github.io/spec/core/text/instructions.html#expressions

Note: this will just be an alias of `parseInstructions(fromIndex)`

### PR DESCRIPTION
**Pull request by jasonwyatt**
Fixes #40

### CODE REVIEW

File: library/src/main/java/kwasm/format/text/expressionTest.kt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ */
+
+package kwasm.format.text
+
+import kwasm.ast.Expression
+import kwasm.format.text.token.Token
+
+/** Parses an [Expression] from the receiving [List] of [Token]s. */
+fun List<Token>.parseExpression(fromIndex: Int): ParseResult<Expression> =
+    parseInstructions(fromIndex).let { ParseResult(Expression(it.astNode), it.parseLength) }
```
Comment (welnanick) [inline library/src/main/java/kwasm/format/text/expressionTest.kt]: Is this file named right?

Comment (jasonwyatt) [reply library/src/main/java/kwasm/format/text/expressionTest.kt]: Oh crap, thanks.

File: .idea/kotlinc.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (welnaseth) [inline .idea/kotlinc.xml]: should this file be here?

File: library/src/main/java/kwasm/format/text/expressionTest.kt (hunk 1 above)
Comment (jasonwyatt) [reply library/src/main/java/kwasm/format/text/expressionTest.kt]: Fixed.

File: .idea/kotlinc.xml (hunk 2 above)
Comment (jasonwyatt) [reply .idea/kotlinc.xml]: I honestly don't know...  let's leave it for now, and delete it if it causes a problem

**Pull request merged**